Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos

```diff
@@ -411,6 +411,7 @@ convertModifiers("Modelica.StateGraph.Temporary.NumericValue",
                   {"hideConnector"}, {"use_numberPort=not %hideConnector%"})
 
 convertModifiers({"Modelica.StateGraph.Interfaces.PartialStep",
+                  "Modelica.StateGraph.InitialStep",
```

Comment:
That lin is redundant.

Reply:
Why?

Reply:
Oops. I must have made an error. I'll check

Reply:
Same as line 415.

Reply:
Ah didn't have the latest version check out. Yes no reason to have it twice.

Reply:
It should now be fixed (but not yet squashed). I must have had some unrelated change in that file for unknown reason.